frontend/src/app/features/leadership-experiences/leadership-experience.model.ts

```diff
@@ -0,0 +1,9 @@
+import { MemberModel } from '../member/member.model';
+import { LeadershipExperienceTypeModel } from './leadership-experiences-type/leadership-experience-type.model';
+
+export interface LeadershipExperienceModel {
+  id: number;
+  member: MemberModel;
+  leadershipExperienceType: LeadershipExperienceTypeModel;
```

Comment:
```suggestion
  experience: LeadershipExperienceTypeModel;
```
I think this has also been renamed.